### PR TITLE
Radio button error fix

### PR DESCRIPTION
**Pull request by DevYuns**
## Description

Fixed errors below

1. It can't be rendered in IOS, Android 
2. Radio border can't be rendered in web

## Checklist

Before you create this PR confirms that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide](https://github.com/dooboolab/dooboo-ui/blob/master/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] Run `yarn test` or `yarn test -u` if you need to update snapshot.
- [x] Run `yarn lint`
- [x] I am willing to follow-up on review comments in a timely manner.

### CODE REVIEW

File: main/RadioButton/index.tsx [hunk 1]
```diff
@@ -56,12 +56,9 @@ const SCInputRow = styled.TouchableOpacity<IInputRowProps>`
   padding: 8px 0;
 `;
 
-const SCOuterCircle = styled.View.attrs(
-  ({ borderRadius, borderWidth }: ICircleProps) => ({
-    borderRadius,
-    borderWidth,
-  }),
-)<ICircleProps>`
+const SCOuterCircle = styled.View<ICircleProps>`
```
Comment (devethan) [inline main/RadioButton/index.tsx:59]: `SCOuterCircle` should be changed for better readable.
The same is `SCInputRow`.
I think there's need to re-define of `SC`.

File: main/RadioButton/index.tsx [hunk 2]
```diff
@@ -56,12 +56,9 @@ const SCInputRow = styled.TouchableOpacity<IInputRowProps>`
...
+  border-radius: ${({ borderRadius }): number => borderRadius};
```
Comment (devethan) [inline main/RadioButton/index.tsx]: Is there no problem even if you don't provide the default value?

**Review by devethan — CHANGES_REQUESTED** [review]

File: main/RadioButton/index.tsx (hunk 2 above)
Comment (DevYuns) [reply main/RadioButton/index.tsx]: It already has default value. Could you check the ' line:34/const DEFAULT CIRCLE' ?

File: main/RadioButton/index.tsx (hunk 1 above)
Comment (DevYuns) [reply main/RadioButton/index.tsx:59]: I'll consider it. :)

File: main/RadioButton/index.tsx (hunk 2 above)
Comment (DevYuns) [reply main/RadioButton/index.tsx:60]: I found an additional issue. when 'InnerCircleAnim' runs, the animation value isn't placed on center.

**Comment by DevYuns** [comment]
I found an additional issue. when 'InnerCircleAnim' runs, the animation value isn't placed on center.

**Comment by devethan** [comment]
> I found an additional issue. when 'InnerCircleAnim' runs, the animation value isn't placed on center.

only web

**Review by hyochan — COMMENTED** [review]
I'll take to look on this

**Comment by hyochan** [comment]
Dg66sFvKlZ


Done? Looks like a simple math problem.

**Review by hyochan — APPROVED** [review]
Looks like we need to refactor this component to make it easier to use. Let's not take anymore time with this.

**Pull request merged**